### PR TITLE
[PATCH v2] test: performance: add pktio slow path function benchmark application

### PR DESCRIPTION
**Pull request by MatiasElo**

### CODE REVIEW

File: test/performance/bench_common.h [hunk 1]
```diff
@@ -108,10 +108,131 @@ void bench_suite_init(bench_suite_t *suite);
...
 /**
- * Run tests suite and print results
+ * Run test suite and print results
+ *
+ * The argument is of type 'bench_suite_t *'.
+ *
+ * @retval 0 on success
+ * @retval <0 on failure
  */
 int bench_run(void *arg);
```
Comment (psavol) [inline test/performance/bench_common.h:118]: Doxygen tags (/**, @retval, ...) could be removed from comments as this file does not seem to be part of doxygen generated documentation (no doxygen errors from missing @param).

File: test/performance/odp_bench_pktio_sp.c [hunk 2]
```diff
@@ -0,0 +1,963 @@
...
+		exit(EXIT_FAILURE);
+	}
+
+	/* Init this thread */
+	if (odp_init_local(instance, ODP_THREAD_CONTROL)) {
+		ODPH_ERR("Local init failed\n");
+		exit(EXIT_FAILURE);
+	}
+
+	ret = (odp_schedule_config(NULL));
```
Comment (psavol) [inline test/performance/odp_bench_pktio_sp.c:867]: extra brackets around function call

Comment (MatiasElo) [reply test/performance/odp_bench_pktio_sp.c]: Fixed in V2.

File: test/performance/bench_common.h (hunk 1 above)
Comment (MatiasElo) [reply test/performance/bench_common.h:115]: Doxygen format removed in V2.

**Review by psavol — APPROVED** [review]

**Pull request merged**